api/lccs-ws.yaml

```diff
@@ -244,6 +244,22 @@ paths:
           $ref: '#/components/responses/ServerError'
 
   /classification_systems/style_formats/{style_format_id}:
```

Comment:
Is it ok?
According the LCCS-DB, there is no relationship between a classification system and style_formats directly.

What is the purpose of this resource?